ElectronClient/app/gui/NotebookPropertiesDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+
+const { _ } = require('lib/locale.js');
+const { themeStyle } = require('../theme.js');
+const DialogButtonRow = require('./DialogButtonRow.min');
+
+const Folder = require('lib/models/Folder.js');
+
+interface NotebookPropertiesDialogProps {
+	theme: number,
+	folderId: string,
+	onClose: Function,
+}
+
+export default function NotebookPropertiesDialog(props: NotebookPropertiesDialogProps) {
+	console.info('Render NotebookPropertiesDialog');
+
+	const [notebook, setNotebook] = useState<any>({});
```

Comment:
Please clearly name what representation of a folder this is. In this case, it's the form own representation (it only needs to have an ID, title and icon) so name it "formFolder" so that we know. When it's just "folder" it means it's the representation as it is in the database.